### ISSUE TITLE
remove "when weapon is active" from pretty boy's pocket pistol stat description

### ISSUE THREAD

**Issue by vlad55523** [mastercomfig/team-comtress-2#513]
### Description

the "when weapon is active" part from the pretty boy pocket pistol's stats was left there from an earlier version which negated fall damage while weapon is active. none of the stats are affecting the player whether the weapon is active  or not with this version, so it should be removed to avoid confusion.

### Checklist

- [ ] None of the open *or* closed issues document this idea.
- [ ] This is a new idea, not a bug.
- [ ] This idea does not seriously affect game balance.
- [ ] This is obviously an improvement, not something that people commonly disagree on.
- [ ] This is not an idea for new game content (e.g. new weapons, maps, sounds).

- [x] This idea is for the code of the game, not for textures, sounds, or other resources.
- [ ] This idea has been implemented in vanilla TF2.
- [ ] There is proof that this idea is well liked (e.g. a history of use of a similar mod in a competitive league).
- [ ] This idea has been implemented in a mod. 
- [ ] This feature has existed in the vanilla game but was removed.
- [ ] This idea is for internal procedures or tooling (e.g. improving the build process, supporting new systems).

### Screenshots

TC2 screenshot:
![20201111121725_1](https://user-images.githubusercontent.com/71296104/98800420-4754a500-2408-11eb-8fd3-e9d6048672d2.jpg)
TF2 screenshot: 
![20201111121825_1](https://user-images.githubusercontent.com/71296104/98800541-75d28000-2408-11eb-82d3-6bf0a1986ab7.jpg)

### Alternatives

1. 
2. 
3.

**Comment by shanepm** [comment]
This sort of change is difficult as, to my knowledge, we're pulling the production item schema, so valve would need to change it

**Comment by MeDerpWasTaken** [comment]
Isn't this only fixable by valve?

**Comment by agrastiOs** [comment]
@MeDerpWasTaken No.

**Comment by MeDerpWasTaken** [comment]
> @MeDerpWasTaken No.

can't we not edit the schematics?

**Comment by agrastiOs** [comment]
> 
> 
> > @MeDerpWasTaken No.
> 
> can't we not edit the schematics?

that's correct that we are currently using the production item schema, however, mastercoms has mentioned that item schema issues aren't really out of scope, so we could possibly have a solution to implement item schema fixes in the future.

**Comment by shanepm** [comment]
yeah, it's just we'd need to disable pulling the item schema from valve's servers to fix this. is that something we want to do?

**Comment by agrastiOs** [comment]
@shanepm Maybe.
